### PR TITLE
Fix System.ComponentModel.TypeConverter.Tests ConvertTo_NullCulture ListSeparator

### PR DESCRIPTION
**Pull request by AlexRadch**
When Current Culture have List Separator other then coma (,) some tests failed.

### CODE REVIEW

File: src/System.ComponentModel.TypeConverter/tests/Drawing/SizeFConverterTests.cs [hunk 1]
```diff
@@ -153,7 +153,8 @@ public void ConvertTo_NotSupportedException(Type type)
         [Fact]
         public void ConvertTo_NullCulture()
         {
-            Assert.Equal("1, 1", Converter.ConvertTo(null, null, new SizeF(1, 1), typeof(string)));
+            var listSep = CultureInfo.CurrentCulture.TextInfo.ListSeparator;
```
Comment (stephentoub) [inline src/System.ComponentModel.TypeConverter/tests/Drawing/SizeFConverterTests.cs]: Nit: var => string

**Comment by safern** [comment]
@mmitche the test didn't triggered, just one of them did.

**Comment by danmosemsft** [comment]
@dotnet-bot test this please

**Comment by AlexRadch** [comment]
Why "Innerloop Ubuntu14.04 Release Build and Test" fail? I can not open details page.

**Comment by stephentoub** [comment]
> Should I replace var on string?

Yes, please make those changes I mentioned.

> Why "Innerloop Ubuntu14.04 Release Build and Test" fail? I can not open details page.

We were having CI problems yesterday, and the system appears to be entirely down now (@mmitche).  I don't believe the failure is related to your PR.

**Comment by AlexRadch** [comment]
> > Should I replace var on string?
> Yes, please make those changes I mentioned.
I see, but I can not translate "Nit:" word. What does it mean?

**Comment by stephentoub** [comment]
> I see, but I can not translate "Nit:" word. What does it mean?

Ah, sorry... we use it to mean "something very minor".

**Comment by AlexRadch** [comment]
I replaced var on string

**Comment by stephentoub** [comment]
@dotnet-bot test this please

**Pull request merged**